test/unit/test.discovery.v1.js

```diff
@@ -552,7 +552,7 @@ describe('discovery-v1', function() {
             paths.federatedquery +
             '?version=' +
             service.version + // query string params order changed, shouldn't be a problem for the service...
-              '&collection_ids=%5Bcol1-guid%2Ccol2-guid%5D&filter=yesplease&natural_language_query=a%20question%20about%20stuff%20and%20things&count=10&sort=%2Bfield_1%2C-field_2'
+              '&collection_ids=col1-guid%2Ccol2-guid&filter=yesplease&natural_language_query=a%20question%20about%20stuff%20and%20things&count=10&sort=%2Bfield_1%2C-field_2'
```

Comment:
Can we check that more than 1 collection id can be query at the same time? I'm worried about this change